publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RMQHelper.java

```diff
@@ -77,8 +79,9 @@ public void rabbitMqPropertiesInit(String protocol) {
     /***
```

Comment:
format, not our change but please comment the part correctly in this document. /** is the correct indentation